src/core/best-practices.md

```diff
@@ -155,3 +155,60 @@ You can use our [Annotation Plugin](/core/software/pupil-capture/#annotations) t
 the recording on desired events, such as trigger events. Annotations can be sent via the keyboard or programmatically. 
 [This script](https://github.com/pupil-labs/pupil-helpers/blob/master/python/remote_annotations.py) demonstrates how you 
 can send remote annotations over the network.
+
+## Pupillometry
+
+[Pupillometry](https://doi.org/10.1002/wcs.1323) is the study of temporal changes in pupil diameter in response to 
+external light stimuli and/or cognitive processing. Pupil Core reports pupil diameter in mm provided by the 
+[pye3d](/developer/core/pye3d/#pye3d-pupil-detection) model: `diameter_3d`, and in pixels as observed in the eye 
+videos: `diameter`. 
+
+Pupil size in pixels is dependent on the camera to pupil distance, and is not corrected for perspective. 
+The output of pye3d more accurately reflects pupil size and will thus be preferable for most users.
+
+### Camera to Pupil Distances 
```

Comment:
In the section below, we might want to differentiate more clearly between
- eye-camera-to-_eye_ and
- eye-camera-to-_pupil_

differences. The former is just a proxy for the latter but is controllable. The latter depends on the former but includes a gaze angle dependency which is not necessarily controllable.